templates/repo/issue/labels/label_edit_modal.tmpl

```diff
@@ -50,7 +50,7 @@
 			<div class="field">
 				<label for="color">{{ctx.Locale.Tr "repo.issues.label_color"}}</label>
 				<div class="column js-color-picker-input">
-					<input name="color" value="#70c24a" placeholder="#c320f6" required maxlength="7">
+					<input name="color" value="#70c24a" placeholder="#c320f6" required pattern="^#?(?:[A-Fa-f0-9]{3}|[A-Fa-f0-9]{6})$" maxlength="7">
```

Comment:
why `?:`? I think it could be simplified to `"^#?([\dA-Fa-f]{3}|[\dA-Fa-f]{6})$" ?